qiskit_machine_learning/neural_networks/estimator_qnn.py

```diff
@@ -131,6 +131,9 @@ def _preprocess(
 
     def _forward_postprocess(self, num_samples: int, result: EstimatorResult) -> np.ndarray:
         """Post-processing during forward pass of the network."""
+        print('====',result.values)
```

Comment:
We will need to remove the print right - maybe unintentionally checked in like this